backend/api/Startup.cs

```diff
@@ -37,7 +37,9 @@
 using Mapster;
 using Pims.Api.Helpers.Mapping;
 using System.Collections.Generic;
+using DocumentFormat.OpenXml.EMMA;
```

Comment:
According to https://www.nuget.org/packages/DocumentFormat.OpenXml/ this is used for working with Office Word, Excel, and PowerPoint documents. 

Do we need it to enable HTTPS and JWT tokens on Swagger??

Reply:
Configuring authentication on Swagger does allow for a more complete tool/documentation.